tests/s3/test_object_tagging.py

```diff
@@ -1387,3 +1388,380 @@ def test_get_non_existing_object_tagging_2487(self):
+            self.bucket_name,
+            self.object_name,
+            tag)
+        assert (resp[0]), resp[1]
+        self.log.info("Created multipart upload with tag key having valid special characters")
+        self.log.info("Performing list multipart uploads")
+        resp = self.s3_mp_obj.list_multipart_uploads(
+            self.bucket_name)
+        assert resp[0], resp[1]
+        upload_id = resp[1]["Uploads"][0]["UploadId"]
+        self.log.info(
+            "Performed list multipart upload on a bucket %s",
+            self.bucket_name)
+        self.log.info("Uploading parts to a bucket %s",
+                      self.bucket_name)
+        resp = self.s3_mp_obj.upload_parts(
+            upload_id,
+            self.bucket_name,
+            self.object_name,
+            S3_OBJ_TST["test_9418"]["single_part_size"],
+            total_parts=S3_OBJ_TST["test_9418"]["total_parts"],
+            multipart_obj_path=self.file_path)
+        assert resp[0], resp[1]
+        upload_parts_list = resp[1]
+        self.log.info(
+            "Parts are uploaded to a bucket %s",
+            self.bucket_name)
+        self.log.info(
+            "Performing list parts of object %s",
+            self.object_name)
+        resp = self.s3_mp_obj.list_parts(upload_id,
+                                         self.bucket_name,
+                                         self.object_name)
+        assert resp[0], resp[1]
+        self.log.info("Performed list parts operation")
+        self.log.info(
+            "Performing complete multipart upload on a bucket %s",
+            self.bucket_name)
+        resp = self.s3_mp_obj.complete_multipart_upload(
+            upload_id,
+            upload_parts_list,
+            self.bucket_name,
+            self.object_name)
+        assert resp[0], resp[1]
+        self.log.info(
+            "Performed complete multipart upload on a bucket %s",
+            self.bucket_name)
+        self.log.info("Retrieving tag of an object %s",
+                      self.object_name)
+        resp = self.tag_obj.get_object_tags(
+            self.bucket_name,
+            self.object_name)
+        assert (resp[1] == {S3_OBJ_TST["test_9429"]["key"]: S3_OBJ_TST["test_9429"]["value"]}), \
+            resp[1]
+        self.log.info("Retrieved tag of an object")
+        self.log.info("Verify Multipart Upload with tag key having valid special characters")
+
+    @pytest.mark.s3_ops
+    @pytest.mark.s3_object_tags
+    @pytest.mark.tags("TEST-42776")
+    def test_mpupload_with_tagvalue256chars_42776(self):
+        """Verify Multipart Upload with tag having
+        tag values up to 256 Unicode characters in length"""
+        self.log.info("Verify Multipart Upload with tag having "
+                      "tag values up to 256 Unicode characters in length")
+        self.log.info("Creating a bucket with name %s",
+                      self.bucket_name)
+        resp = self.s3_test_obj.create_bucket(
```

Comment:
We follow line length limit up to 100 characters, can you please follow same to reduce number of lines from this test module?